grails-app/controllers/org/pih/warehouse/invoice/InvoiceController.groovy

```diff
@@ -68,14 +68,14 @@ class InvoiceController {
         def invoiceInstance = Invoice.get(params.id)
         if (invoiceInstance) {
             try {
-                invoiceDataService.delete(invoiceInstance.id)
+                invoiceService.deleteInvoice(invoiceInstance)
```

Comment:
why was that the issue?

Reply:
I guess as a short-term solution it is an okay solution but would be good if there was some sort of annotation handling that case (like join for gets) or perhaps g3 has better working cascade deletes for this than it was on g1.